tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +109,21 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result 
 
-@pytest.mark.skip
+    assert len(tai.inventory) == len(jesse.inventory)
```

Comment:
What if before the swap the length of their inventory lists were not the same? 

This assertion would fail. Are we actually concerned with testing that their lists are the same length?

We should be explicit about what we want to test: the length of Tai's inventory list did not change after the swap and the length of Jesse's inventory list did not change after the swap. 

Prefer to write these assertions like:
```py
assert len(tai.inventory) == 3
assert len(jesse.inventory) == 3

Reply:
✅ Fixed (will push later!)